app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/SubscriptionConnector.scala

```diff
@@ -36,11 +38,20 @@ class SubscriptionConnector @Inject() (
   metrics: Metrics
 )(implicit ec: ExecutionContext) {
 
+  private val logger = Logger(this.getClass)
+
   def get(
     pptReferenceNumber: String
   )(implicit hc: HeaderCarrier): Future[SubscriptionDisplayResponse] = {
     val timer = metrics.defaultRegistry.timer("ppt.subscription.get.timer").time()
     httpClient.GET[SubscriptionDisplayResponse](appConfig.pptSubscriptionUrl(pptReferenceNumber))
+      .map {
+        response =>
+          logger.info(
+            s"Retrieved subscription for ppt reference number [$pptReferenceNumber] had response [${toJson(response)}]"
```

Comment:
There's too much in this particular response. I'd just log the ppt ref here.

Reply:
Done